integration_tests/src/main/python/delta_lake_auto_compact_test.py

```diff
@@ -113,13 +131,16 @@ def test_auto_compact_partitioned(spark_tmp_path, auto_compact_conf, enable_dele
     def read_data(spark, table_path):
         return spark.read.format("delta").load(table_path).orderBy("id", "part")
 
+    conf_enable_auto_compact = copy_and_update(_conf, {auto_compact_conf: "true"})
+
     assert_gpu_and_cpu_writes_are_equal_collect(
         write_func=write_to_delta(enable_deletion_vectors, is_partitioned=True),
         read_func=read_data,
         base_path=data_path,
-        conf=_conf)
+        conf=conf_enable_auto_compact)
 
     def read_metadata(spark, table_path):
```

Comment:
I know it's not a part of this PR but can we refactor this method into a single one and reuse it? I am fine with you doing it as a follow-on

Reply:
I'm going to merge this PR now, but I will do it as a follow-on once you clarify it.